source/manual/datalabs-reference-info.html.md

```diff
@@ -0,0 +1,310 @@
+
+### Assurance throughout development
+
+Quality assurance considerations should be taken into account throughout the life cycle of the analysis and not just at the end. Effective communication is crucial when understanding the problem, designing the analytical approach, conducting the analysis and relaying the outputs.
+
+### Verification and validation
+
+Analytical quality assurance is more than checking that the analysis is error-free and satisfies its specification (verification). It must also include checks that the analysis is appropriate, i.e. fit for the purpose for which it is being used (validation).
+
+### Analysis with RIGOUR
+
+Quality analysis needs to be the following:
+
+- repeatable (R)
+- independent (I)
+- grounded in reality (G)
+- objective (O)
+- have understood and managed uncertainty (U)
+- the results should address the initial question robustly (R)
+
+In particular, it is important to accept that uncertainty is inherent within the inputs and outputs of any piece of analysis. It is important to establish how much we can rely upon the analysis for a given problem.
+
+These principles must be considered when undertaking any work involving data/models. When setting up a project, use `govcookiecutter` to help concurrently set up all AQA documentation. Note this only sets up the documentation. You still need to perform the AQA itself.
+
+Note that AQA is not just about software quality assurance. It can also include dealing with ethical considerations, reasons for choosing the method/technique, and validating analytical assumptions and caveats.
+
+### Further information
+
+[Additional Aqua book resources are available](https://www.gov.uk/government/collections/aqua-book-resources), and the Government Analytical Function, Government Data Quality Hub, and other departments have also produced:
+
+- guides to ensure your [work is fit for purpose when working to very tight deadlines](https://www.gov.uk/government/publications/urgent-data-quality-assurance-guidance)
+- guides to ensure your [data is fit for purpose](https://www.gov.uk/government/publications/the-government-data-quality-framework)
+- a [curriculum around quality assurance, validation, and data linkage](https://www.gov.uk/guidance/af-learning-curriculum-technical#QualityAssuranceAF)
+
+## GOV.UK developer documentation
+
+See the [documentation on document types on GOV.UK](https://docs.publishing.service.gov.uk/document-types.html) for information on the various document types present on GOV.UK. This list may be incomplete.
+
+See the [documentation on content schemas](https://docs.publishing.service.gov.uk/content-schemas.html) for more information on schema used for different content items on the Content Store.
+
+This page is useful if you are working with the Content Store as it tells you what fields are available for different content document types again. This list may be incomplete.
+
```

Comment:
I'd be tempted to add in a section about this chrome plugin for viewing the JSON of GOV.UK pages, it also lets you view A and B versions of pages

## Viewing the JSON of a GOV.UK page 
You can also insert `/api/content` into the url to view e.g. `https://www.gov.uk/api/content/browse/benefits/disability` or use the [GOV.UK Google Chrome extension](https://github.com/alphagov/govuk-browser-extension)